packages/packages.csv

```diff
@@ -68,7 +68,7 @@ Microsoft.AspNetCore.Mvc.ApiExplorer,ship
 Microsoft.AspNetCore.Mvc.Core,ship
 Microsoft.AspNetCore.Mvc.Cors,ship
 Microsoft.AspNetCore.Mvc.DataAnnotations,ship
-Microsoft.AspNetCore.Mvc.Dnx,ship
+Microsoft.AspNetCore.Mvc.Dnx,noship
```

Comment:
@Eilon told me we don't want to check the MVC PR just yet, so I'll just go ahead and merge this one.